src/test/java/edu/hm/hafner/analysis/parser/GrypeParserTest.java

```diff
@@ -0,0 +1,40 @@
+package edu.hm.hafner.analysis.parser;
+
+import edu.hm.hafner.analysis.AbstractParserTest;
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.Report;
+import edu.hm.hafner.analysis.Severity;
+import edu.hm.hafner.analysis.assertions.SoftAssertions;
+
+class GrypeParserTest extends AbstractParserTest {
+    protected GrypeParserTest() {
+        super("grype-report.json");
+    }
+
+    @Override
+    protected void assertThatIssuesArePresent(final Report report, final SoftAssertions softly) {
+        softly.assertThat(report).hasSize(29).hasDuplicatesSize(0);
+        softly.assertThat(report.get(0))
+                .hasFileName("tomcat-jdbc/8.0.28/tomcat-jdbc-8.0.28.jar")
+                .hasSeverity(Severity.WARNING_NORMAL)
+                .hasCategory("Medium")
+                .hasType("CVE-2015-5345")
+                .hasMessage(
+                        "The Mapper component in Apache Tomcat 6.x before 6.0.45, 7.x before 7.0.68, 8.x before 8.0.30, and 9.x before 9.0.0.M2 processes redirects before considering security constraints and Filters, which allows remote attackers to determine the existence of a directory via a URL that lacks a trailing / (slash) character.")
+                .hasDescription("https://nvd.nist.gov/vuln/detail/CVE-2015-5345");
```

Comment:
I'm not sure if the plain link will be clickable in the UI. See https://github.com/jenkinsci/analysis-model/blob/533ac7e8b6803b617edd247da507eedc27c3e6aa/src/main/java/edu/hm/hafner/analysis/parser/ErrorProneParser.java#L72
on how to create an HTML link.

Reply:
Done, the link is clickable now. Thanks!